README.md

```diff
@@ -1,54 +1,95 @@
 # wavefront-opentracing-bundle-java
 
-The Wavefront OpenTracing Bundle builds a fat-jar containing the [Wavefront by VMware OpenTracing
- SDK](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java) and a `TracerFactory` 
- implementation.
-
-Tracer parameters can be configured via a `tracer.properties` configuration file:
-
-```properties
-# Required application tags
-wf.application=myApp
-wf.service=myService
-
-# Optional application tags
-wf.cluster=us-west
-wf.shard=primary
-
-# Reporting through direct ingestion
-wf.reportingMechanism=direct
-wf.server=<replace-with-wavefront-url>
-wf.token=<replace-with-wavefront-api-token>
-
-# Reporting with a Wavefront proxy
-#wf.reportingMechanism=proxy
-#wf.proxyHost=<replace-with-wavefront-proxy-hostname>
-#wf.proxyMetricsPort=2878
-#wf.proxyDistributionsPort=2878
-#wf.proxyTracingPort=30000
-```
-
-Parameters can be configured or overridden through System properties as well:
-
-```bash
-java -cp:$MYCLASSPATH:wavefront-opentracing-bundle-java.jar \
-    -Dwf.service=MyOtherService \
-    com.mycompany.MyOtherService
-```
-
-Parameters can also be configured via YAML files. Two YAML files would be needed -- one to 
-[configure application tags](https://github.com/wavefrontHQ/wavefront-jersey-sdk-java#1-configure-application-tags) 
-and one to [configure Wavefront reporting](https://github.com/wavefrontHQ/wavefront-jersey-sdk-java#2-configure-wavefront-reporting). 
-The paths of these YAML files should be specified in `tracer.properties` or as System properties:
-
-```bash
-java -cp:$MYCLASSPATH:wavefront-opentracing-bundle-java.jar \
-    -Dwf.applicationTagsYamlFile=application-tags.yaml \
-    -Dwf.reportingConfigYamlFile=wf-reporting-config.yaml \
-    com.mycompany.MyService
-```
-
-Note that parameters configured via `tracer.properties` or System properties would override parameters configured via YAML files.
+Welcome to the Wavefront Java Tracing Agent! 
+
+* [Prerequisites](#Prerequisites)
+* [Configuring the Parameters](#Configuring-the-Parameters)
+* [Parameters](#Parameters)
+* [Instrumenting a Java Based Container](#Instrumenting-a-Java-Based-Container)
+
+The Wavefront Java Tracing Agent provides application observability without having to do any code changes.
+
+<p align="left">
+  <img src="/docs/wavefront_java_tracing_agent.png">
+</p> 
+
+The Wavefront Java Tracing Agent includes:
+* The [Java OpenTracing SpecialAgent](https://github.com/opentracing-contrib/java-specialagent), which automatically instruments Java applications with traces.
+* The Wavefront OpenTracing Bundle that is implemented in this repository, which sends the tracing data to Wavefront for observability.
+ 
+## Prerequisites
+
+*  Java 7 or above.
+* [Download](https://github.com/opentracing-contrib/java-specialagent#2111-stable) the latest version of the Java OpenTracing SpecialAgent to your application's directory.
+
+## Configuring the Parameters
```

Comment:
rename this since it includes more than just configuring the parameters now. perhaps `Usage`

Reply:
Good point! I will use Setup steps.. That's sort of the same format the SDKs use...